Samples/3f-example-generation/shell/stderr.txt

```diff
@@ -1,6 +1,26 @@
+  at ...
+  at ...
```

Comment:
@dsgouda uh I think there were some errors during generation, always look at the diff ;)